provisioning/tasks/cron.yml

```diff
@@ -9,5 +9,6 @@
     month: "{{ item.month | default('*') }}"
     job: "{{ item.job }}"
     state: "{{ item.state | default('present') }}"
+    user: "{{ item.user | default(ansible_user) }} "
```

Comment:
Can you set this to:

```yaml
    user: "{{ item.user | default(omit) }}"
```

So it just doesn't set it if none is provided? `ansible_user` can do... weird things sometimes.